tools/openblas_support.py

```diff
@@ -77,7 +77,7 @@ def download_openblas(target, plat, ilp64):
         suffix = f'manylinux{ml_ver}_{arch}.tar.gz'
         typ = 'tar.gz'
     elif plat == 'macosx-x86_64':
-        suffix = 'macosx_10_9_x86_64-gf_1becaaa.tar.gz'
+        suffix = 'macosx_10_11_x86_64-gf_dd3e148.tar.gz'
```

Comment:
I think this OpenBLAS change means that you have to set `MACOSX_DEPLOYMENT_TARGET=10.11`. One needs to be making `numpy-1.24.0.dev0-cp39-cp39-macosx_10_11_x86_64.whl` instead of `numpy-1.24.0.dev0-cp39-cp39-macosx_10_9_x86_64.whl`.
Using https://github.com/MacPython/gfortran-install/pull/11/files may still allow `macosx_10_9_x86_64-gf_1becaaa.tar.gz` to be used, but that would require experimentation.